action.yaml

```diff
@@ -195,12 +195,16 @@ runs:
             ')' \
             -print0)
 
+        perm_executable="/111"
+        # BSD find on macOS does not support the slash syntax for permissions.
+        [ "$(uname -s)" = "Darwin" ] && perm_executable="+111"
```

Comment:
This suggestion would have been valid in more general contexts, but this repository is a GitHub action. It will only ever run in a GitHub Actions environment, which is either Ubuntu Linux, macOS, or Windows. Since shellcheck isn't useable on Windows, our remaining target platforms are only Ubuntu and macOS.